example/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:4.4.4
+ENV PATH ./node_modules/.bin/:$PATH
+ENV NPM_CONFIG_LOGLEVEL warn
```

Comment:
By default the official node image sets this to verbose, which produces way too many logs. Set it back to warn (default for npm when you install it on os x)